src/posts/paper/14-research-sources.ts

```diff
@@ -163,7 +163,9 @@ Another important thing to know about the infectious period is that only about 9
 
 ![He et al](${heExtended})
 
-(If you were previously familiar with this source, note that it was substantially updated in a [07 Aug 2020 author correction](https://www.nature.com/articles/s41591-020-1016-z).)
+*(If you were previously familiar with this source, note that it was substantially updated in a [07 Aug 2020 author correction](https://www.nature.com/articles/s41591-020-1016-z).)*
```

Comment:
Did you intend to italicize this?

Reply:
Yeah. I like sidenotes like this to be in italics. But now that I think about it, it's actually better as a foot note.

Reply:
Okay I moved it to a footnote.